README.md

```diff
@@ -19,7 +19,7 @@ Elevate is a JSON parsing framework that leverages Swift to make parsing simple,
 ## Requirements
 
 - iOS 8.0+ / Mac OS X 10.10+ / tvOS 9.0+ / watchOS 2.0+
-- Xcode 8.0 beta 6+
+- Xcode 8.0
```

Comment:
I did the following in AF and Willow.

```
- iOS 9.0+ / macOS 10.11+ / tvOS 9.0+ / watchOS 2.0+
- Xcode 8.0+
- Swift 3.0+
```

If you haven't actually set the deployment targets to that yet, definitely make sure to do that.